src/gluonts/transform/convert.py

```diff
@@ -27,6 +26,82 @@
 )
 
 
+def erf(x: np.array) -> np.array:
```

Comment:
This one seems to have only one specific usage throughout the code and I think one could argue that it should be a function for the class that uses it, no? Not sure about other cases here.

Reply:
I think it's fine to have them as standalone functions, since `erf` isn't tied to some object state.